feature/gnmi/ate_tests/telemetry_basic_check_test/telemetry_basic_check_test.go

```diff
@@ -395,40 +395,35 @@ func TestComponentParent(t *testing.T) {
 
 	for _, tc := range cases {
 		t.Run(tc.desc, func(t *testing.T) {
-			t.Logf("Found card list for Component Type %v : %v", tc.componentType, cardList[tc.desc])
-			if len(cardList[tc.desc]) == 0 {
-				t.Fatalf("Get Card list for %q: got 0, want > 0", dut.Model())
+			t.Logf("Found Component list for Type %v : %v", tc.componentType, compList[tc.desc])
+			if len(compList[tc.desc]) == 0 {
+				t.Fatalf("Get Component list for %q: got 0, want > 0", dut.Model())
 			}
 			// Validate parent component
-			for _, card := range cardList[tc.desc] {
-				t.Logf("Validate card %s", card)
-				cardName := card
+			for _, comp := range compList[tc.desc] {
+				t.Logf("Validate component %s", comp)
+				compName := comp
+				foundChassis := false
 				for {
-					parent := gnmi.Lookup(t, dut, gnmi.OC().Component(cardName).Parent().State())
+					parent := gnmi.Lookup(t, dut, gnmi.OC().Component(compName).Parent().State())
 					val, present := parent.Val()
 					if !present {
-						t.Fatalf("Parent not present for %q: got %v, want true", card, parent.IsPresent())
+						t.Fatalf("Parent not present for %q: got %v, want true", comp, parent.IsPresent())
 					} else {
```

Comment:
no need to put this logic in an else-lock, since the !present case always exits the loop